src/kernel/src/machine/arm/virt/info.rs

```diff
@@ -0,0 +1,103 @@
+use fdt::Fdt;
```

Comment:
Each file needs a header, what it is, why it's there, who wrote it. Look at any .c file in FreeBSD to see how we do this.